ocean_lib/models/side_staking.py

```diff
@@ -45,26 +58,6 @@ def get_vesting_last_block(self, datatoken: str) -> int:
     def get_vesting_amount_so_far(self, datatoken: str) -> int:
         return self.contract.caller.getvestingAmountSoFar(datatoken)
 
-    def can_stake(self, datatoken: str, amount: int) -> bool:
-        return self.contract.caller.canStake(datatoken, amount)
-
-    def stake(self, datatoken: str, amount: int, from_wallet: Wallet) -> str:
-        return self.send_transaction("Stake", (datatoken, amount), from_wallet)
-
-    def can_unstake(self, datatoken: str, liquidity_pool_token_in: int) -> bool:
-        return self.contract.caller.canUnStake(datatoken, liquidity_pool_token_in)
-
-    def unstake(
-        self,
-        datatoken: str,
-        dt_amount_in: int,
-        pool_amount_out: int,
-        from_wallet: Wallet,
-    ) -> str:
-        return self.send_transaction(
-            "UnStake", (datatoken, dt_amount_in, pool_amount_out), from_wallet
-        )
-
```

Comment:
I see why these methods were removed -- they're more of an interface for the Pool contract, and didn't need to be exposed.

However why were they deleted here? It's out of scope of this issue. It would have made more sense to have a separate issue to do that. Keep the scope of each issue - and related PR - tight and well-defined.

That said: rather than adding work, this time just keep it part of this PR. In the future, please keep scope tight.

Reply:
I have created #766 for this particular issue. I will revert the changes here.